bin/citgm-all.js

```diff
@@ -47,6 +56,15 @@ if (!app.su) {
   log.warn('root', 'Running as root! Use caution!');
 }
 
+if (app.includeTags){
```

Comment:
If it is by default undefined then I think we need to do some safety checks inside of check-tags to ensure that we are not checking for .length on undefined